engine/language-server/src/test/scala/org/enso/languageserver/websocket/json/LibrariesTest.scala

```diff
@@ -783,20 +753,20 @@ class LibrariesTest extends BaseServerTest {
           }
           """)
 
-      client.expectJson(json"""
-          { "jsonrpc": "2.0",
-            "id": 0,
-            "result": {
-              "availableComponents" : [ ]
-            }
-          }
-          """)
+      val response = client.expectSomeJson()
+      val components = response.hcursor
+        .downField("result")
+        .downField("availableComponents")
+        .as[List[LibraryComponentGroup]]
+        .rightValue
+
+      components should not be empty
```

Comment:
Do we know what `components` will contain? Can we check at least one value inside of it?